src/components/PopupWithForm.js

```diff
@@ -1,11 +1,13 @@
 import Popup from "./Popup.js";
 
 class PopupWithForm extends Popup {
-  constructor(popupSelector, handleFormSubmit) {
+  constructor(popupSelector, handleFormSubmit, defaultSubmitBtnText) {
     super({ popupSelector });
     this._modalForm = this._popupElement.querySelector(".modal__form");
     this._inputList = this._modalForm.querySelectorAll(".modal__input");
     this._handleFormSubmit = handleFormSubmit;
+    this._submitbutton = this._modalForm.querySelector(".modal__save");
+    this._defaultSubmitBtnText = defaultSubmitBtnText;
   }
   close() {
```

Comment:
* You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.